src/plone/restapi/services/users/get.py

```diff
@@ -2,11 +2,14 @@
 from plone.restapi.interfaces import ISerializeToJson
 from plone.restapi.services import Service
 from Products.CMFCore.utils import getToolByName
+from zExceptions import BadRequest
 from zope.component.hooks import getSite
 from zope.component import queryMultiAdapter
 from zope.interface import implements
 from zope.publisher.interfaces import IPublishTraverse
 
+DEFAULT_SEARCH_RESULTS_LIMIT = 25
```

Comment:
@sneridagh this is fine for a first implementation. Though, we have to think about if we either make this configurable, or we add batching and stuff...